packages/components/src/components/accordion/accordionItem.tsx

```diff
@@ -57,6 +57,12 @@ export class IfxAccordionItem {
     return this.open;
   }
 
+  handleSlotChange() { 
+    if(this.open) { 
+      this.openAccordionItem()
+    }
+  }
+
```

Comment:
is there a reason, why this is handled using javascript instead of directly in the css?

Reply:
Kai created this component. The purpose of **this.contentEl.style.maxHeight = `${this.contentEl.scrollHeight}px`;** is to make sure the **max height** equals the height of the content all the time. Setting the **max height** to 100% could cause the content to overflow in some cases.

Reply:
i guess we could do that with css classes too but doesnt matter really